tests/e2e/login/purge.spec.js

```diff
@@ -279,17 +279,21 @@ describe('Purging on login', () => {
     expect(await browser.isElementPresent(reports.reportByUUID(goodFormId2))).toBeTrue;
     expect(await browser.isElementPresent(reports.reportByUUID(badFormId2))).toBeTrue;
 
+    await browser.driver.setNetworkConditions({ offline: true, latency: 0, throughput: 0 });
```

Comment:
How about using https://webdriver.io/docs/api/browser/throttle/ ? eg:

```suggestion
    browser.throttle('offline');
```

Reply:
This is protractor :( :( :(

Reply:
Duh... I'm working on a PR which converts this to wdio and throttles. We can cross that bridge when we come to it...